monkeylib.py

```diff
@@ -140,7 +140,7 @@ def launchmonkey(adb):
         print("fwd->", adb.forward(12345, 12345))
 
         # or 'toybox killall'
-        killres = adb.shell("killall com.android.commands.monkey")
+        killres = str(adb.shell("killall com.android.commands.monkey"))
```

Comment:
Is this because sometimes the result will be None?  in that case, it would be better to change the next line to: if killres and killres.find ....

Reply:
Yup! I did this initially just to get by, but I looked into it more after your comment and it turns out the killall command can return an empty result, but adding -v reports if the signal was sent. In my latest iteration I added that option on top of the killres None check and a regex test.